inspire_matcher/utils.py

```diff
@@ -37,3 +37,55 @@ def get_number_of_author_matches(x_authors, y_authors):
 
     """
     return len(AuthorComparator(x_authors, y_authors).matches)
+
+
+def compute_authors_jaccard_index(x_authors, y_authors):
+    """Return the Jaccard similarity coefficient between 2 author sets.
+
+    Args:
+        x_authors (list(dict)): first schema-compliant list of authors.
+        y_authors (list(dict)): second schema-compliant list of authors.
+
+    Returns:
+        float: Jaccard similarity coefficient between the 2 author sets.
+
+    """
+    intersection_cardinal = get_number_of_author_matches(x_authors, y_authors)
+    union_cardinal = max(len(x_authors), len(y_authors))
```

Comment:
If the record has no authors this will fail right?

Reply:
It shouldn't i believe, but it is called only when there are authors and also the author objects must be schema-compliant.